charts/portworx/templates/serviceaccount-hook.yaml

```diff
@@ -20,12 +20,15 @@ apiVersion: {{ template "rbac.apiVersion" . }}
 metadata:
   annotations:
     "helm.sh/hook-delete-policy": before-hook-creation
-    "helm.sh/hook": "pre-delete,post-delete"
+    "helm.sh/hook": "pre-delete,post-delete,pre-upgrade"
   name: {{ template "px.hookClusterRole" . }}
 rules:
-  - apiGroups: [""]
-    resources: ["nodes"]
-    verbs: ["patch", "get", "update", "list"]
+  # for daemonset to operator migration, we need hooks for all resources deployed by daemonset, due to resources are
+  # different in different helm charts (GCP, IKS, Rancher and portworx), we use wild card here. After daemonset
+  # migration is finished for all customers we shall change this back to limited access.
+  - apiGroups: ["*"]
```

Comment:
We don't know when we can migrate all customers, I feel we might forget here. Can we keep the same permission and change when platform  (GCP, IKS, Rancher and portworx) needed ?